projects/storefrontlib/src/cms-components/myaccount/order/amend-order/amend-order-items/amend-order-items.component.html

```diff
@@ -106,11 +106,18 @@
               }}
             </div>
 
+            <div
+              *ngIf="isConfirmation"
+              class="w-100 text-center cx-order-quantity-value"
+            >
+              {{ getControl(form, item).value }}
+            </div>
             <cx-item-counter
+              *ngIf="!isConfirmation"
               [min]="0"
               [max]="getMaxAmendQuantity(item)"
               [control]="getControl(form, item)"
-              [readonly]="isConfirmation"
+              [class.readonly]="isConfirmation"
```

Comment:
I thought that component takes care of this class?

Reply:
Also ngIf and readonly are negating themselves.

Reply:
Nice catch, thanks!